builder/azure/common/config.go

```diff
@@ -0,0 +1,33 @@
+//go:generate packer-sdc struct-markdown
+
+package common
+
+import (
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+)
+
+const (
+	SkippingImageCreation = "Skipping image creation..."
+)
+
+type Config struct {
+	// Skip creating the image.
+	// Useful for setting to `true` during a build test stage.
+	// Defaults to `false`.
```

Comment:
Should the default be first?

```suggestion
	// Defaults to `false`.
	// Useful for setting to `true` during a build test stage.
```

Reply:
It looks like in the rest of the documentation the defaults is the last thing mentioned. I think we should keep this new one consistent, but would be happy to reorg it if desired by the maintainers.

Reply:
Oh, I just saw it different in some other spots - but def trust you on this one - we can resolve this.